content/workshops/ansible_tower/exercise3.0.adoc

```diff
@@ -0,0 +1,156 @@
+
+[source,bash]
+----
+sudo ansible-galaxy install rhtps.800-53
+----
+
+=== Step 2:
+
+Now that you have the NIST 800-53 role installed on your Tower node, we need to create a playbook that uses it.  First, let's create a storage directory for playbooks, so that we can reference them from a Tower project, and then we'll create the playbook:
+
+[source,bash]
+----
+sudo mkdir -p /var/lib/awx/projects/playbooks
+sudo vim /var/lib/awx/projects/playbooks/800-53.yml
+----
+
+=== Step 3:
+
+Let's create the needed playbook.  You should have a pretty good understanding of playbooks by now, so we won't go over this, in detail.  Please do feel free to ask questions, though, if there is something that you don't understand.
+
+Note the use of the `register`, `with_items`, and `when` directives.  These are how Ansible implements variable creation, recursion, and conditionals, respectively.
+
+[source,bash]
+----
+---
+- hosts: web
+  become: yes
+  vars:
+    scap_reports_dir: /tmp
+  roles:
+    - rhtps.800-53 
+
+  tasks:
+    - name: determine the most recent scap report
+      command: ls -tr /tmp/scap_reports/
+      register: results
+
+    - name: create the scap directory in the web server content
+      file:
+        name: /var/www/html/scap
+        state: directory
+        mode: 0755
+
+    - name: copy SCAP reports to the web server content directory
+      copy:
+        remote_src: True
+        src: "/tmp/scap_reports/{{ item }}"
+        dest: /var/www/html/scap
+        mode: 0644
+      with_items: "{{ results.stdout_lines }}"
+      when: item | match("scan-xccdf-report-*")
+----
+
+=== Step 4:
+
+Save and quit from editing, and then we will move on to the in-Tower setup.
+
+== Configuring 800-53 in Ansible Tower
+
+=== Step 1:
+
+In your Tower window, click on image:at_projects_icon.png[PROJECTS]
+
+=== Step 2:
+
+Select ADD     image:at_add.png[Add,35,25]
+
+=== Step 3:
+
+Enter the following values into your new project.  Note that the `PLAYBOOK DIRECTORY` item should show `playbooks`, as an option, when you click on it.
+
+|===
+|NAME |NIST 800-53
+|DESCRIPTION|NIST 800-53 Project
+|ORGANIZATION|Default
+|SCM TYPE|Manual
+|PROJECT BASE PATH|/var/lib/awx/projects
+|PLAYBOOK DIRECTORY|playbooks
+|===
+
+=== Step 4:
+
+Select SAVE     image:at_save.png[Save,35,25]
+
+=== Step 5:
+
+In your Tower window, click on TEMPLATES
+
+=== Step 6:
+
+Click on ADD image:at_add.png[Add,35,25], and select JOB TEMPLATE
+
+=== Step 7:
+
+Complete the form using the following values.  Note that the PLAYBOOK field should offer `800-53.yml` as an option, when clicked.
+
+|===
+|NAME |NIST 800-53 Job Template
+|DESCRIPTION|Template for NIST 800-53
+|JOB TYPE|Run
+|INVENTORY|Ansible Workshop Inventory
+|PROJECT|NIST 800-53
+|PLAYBOOK|800-53.yml
+|MACHINE CREDENTIAL|Ansible Workshop Credential
+|LIMIT|web
+|OPTIONS
+a|
+- [*] Enable Privilege Escalation
+|===
+
+=== Step 8:
```

Comment:
I would move Step 8 down a line to be the step to run. 

=== Step 8:
Click on the rocketship icon image:at_launch_icon.png[Add,35,25] for the `NIST 800-53 Job Template`.